src/validators/update-todo-item-validator.ts

```diff
@@ -0,0 +1,8 @@
+import { param, ValidationChain } from 'express-validator'
+import { AppContext } from '@typings'
+
+const updateTodoItemValidator = (appContext: AppContext): ValidationChain[] => [
+  param('id', 'VALIDATION_ERRORS.INVALID_ID').isMongoId(),
```

Comment:
Also validate the title in this validator itself.

Reply:
updated